packages/editor/src/components/rich-text/index.js

```diff
@@ -639,6 +639,11 @@ export class RichText extends Component {
 
 				this.onChange( insert( this.getRecord(), toInsert ) );
 			} else {
+				// Some implementations of onSplit might also call onReplace, and
+				// calling onReplace() will destroy the editor, so it's
+				// important that we stop other handlers (e.g. ones
+				// registered by TinyMCE) from also handling this event.
+				event.stopImmediatePropagation();
```

Comment:
Looking at failing e2e tests it seems like this isn't a change that would be expected.

Reply:
Removing this line has fixed the failing e2e tests.